core/src/main/java/io/undertow/server/handlers/encoding/RequestEncodingHandler.java

```diff
@@ -84,7 +85,7 @@ public HttpHandler getNext() {
 
         @Override
         public String name() {
-            return "uncompress";
+            return "uncompress-request";
```

Comment:
I'm rejecting this change because this name is part of Undertow documentation here: http://undertow.io/undertow-docs/undertow-docs-2.0.0/predicates-attributes-handlers.html
While we can assume that nobody was using this handler because it was missing from the HandlerBuilder file, I think we must stick to what the documentation says, at least for version 2.0.x

Reply:
I'm updating this PR since I would like to merge it before tagging the next release. Thanks for the fix btw!